schemas/rum/view-schema.json

```diff
@@ -323,6 +323,26 @@
           },
           "readOnly": true
         },
+        "parent_view": {
```

Comment:
forgot to do like other schema:
```
{
      "$ref": "_parent-view-schema.json"
    },
```

Reply:
Or it could be added in the `_common-schema` ?

Reply:
Done ✔️